TestProjects/UniversalGraphicsTest_Terrain/Assets/Scenes/140_SSAO/Renderers/SSAO_DepthNormalRenderer.asset

```diff
@@ -31,7 +32,7 @@ MonoBehaviour:
   m_GameObject: {fileID: 0}
   m_Enabled: 1
   m_EditorHideFlags: 0
-  m_Script: {fileID: 11500000, guid: de640fe3d0db1804a85f9fc8f5cadab6, type: 3}
```

Comment:
Lot's of GUId changes, is this expected?

Reply:
Yes, ForwardRenderer -> UniversalRenderer